CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [Version 1.3.0](https://github.com/dataiku/dss-plugin-pi-server/releases/tag/v1.3.0) - Feature and bug release - 2025-05-23
+
+- Accept timezone in datetime inputs
+- Add a "Step" column
```

Comment:
Should we have an explanation of what this means, or add a link to the osisoft documentation?